config/features.yml

```diff
@@ -38,12 +38,15 @@ features:
   facility_locator_ppms_legacy_urgent_care_to_pos_locator:
     actor_type: user
     description: force the legacy urgent care path to use the new POS locator
-  facility_locator_ppms_location_query:
+  facility_locator_ppms_use_v1_client:
```

Comment:
Just double checking that you also wanted to change 'facility_locator_ppms_location_query' to 'facility_locator_ppms_use_v1_client'

Reply:
yep, `: facility_locator_ppms_location_query` was removed with its logic and replaced with `: facility_locator_ppms_use_v1_client`

Reply:
actually, this change belongs in a different branch. fixing it